packages/stripe_checkout/lib/src/platforms/stripe_checkout.dart

```diff
@@ -0,0 +1,126 @@
+library stripe_checkout;
+
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+import 'checkout.dart';
+
+Future<CheckoutResponse> redirectToCheckout({
+  required BuildContext context,
+  required String sessionId,
+  required String publishableKey,
+  String? successUrl,
+  String? canceledUrl,
+}) async {
+  final response = await Navigator.of(context).push(
+    MaterialPageRoute(
```

Comment:
For now it is fine but should we enforce the material page here? I think in the future maybe people want to inject there own animations. Still for now it should be fine but let's add it to the backlog

Reply:
Definitely true!